tools/clang/unittests/HLSL/MSFileSysTest.cpp

```diff
@@ -146,12 +146,29 @@ class MockDxcSystemAccess : public IDxcSystemAccess
   }
 
   virtual HRESULT STDMETHODCALLTYPE EnumFiles(LPCWSTR fileName, IEnumSTATSTG** pResult) override {
-    STATSTG items[] =
-    {
-      { L"filename.hlsl", STGTY_STREAM, { 0, 0 }, { 0, 0 }, { 0, 0 }, { 0, 0 }, 0, 0, GUID_NULL, 0, 0 },
-      { L"filename2.fx",  STGTY_STREAM, { 0, 0 }, { 0, 0 }, { 0, 0 }, { 0, 0 }, 0, 0, GUID_NULL, 0, 0 }
-    };
-    unsigned testCount = (unsigned)_countof(items);
+    STATSTG items[] = {{const_cast<wchar_t *>(L"filename.hlsl"),
+                        STGTY_STREAM,
+                        {{0, 0}},
+                        {0, 0},
+                        {0, 0},
+                        {0, 0},
+                        0,
+                        0,
+                        GUID_NULL,
+                        0,
+                        0},
+                       {const_cast<wchar_t *>(L"filename2.fx"),
+                        STGTY_STREAM,
+                        {{0, 0}},
+                        {0, 0},
+                        {0, 0},
+                        {0, 0},
+                        0,
+                        0,
+                        GUID_NULL,
+                        0,
+                        0}};
```

Comment:
Based on my limited intuition for the tool, this seems uncharacteristic of clang-format. Did you clang-format these changes?

Reply:
Yes. That is the result of clang-format :).